docs/snippets/setup-slack-integration.mdx

```diff
@@ -121,6 +121,7 @@ Here is the format in the yml itself:
 slack:
   token: <your_slack_token>
   channel_name: <slack_channel_to_post_at>
+  timezone: <timezone_for_timestamps>
```

Comment:
Maybe we should clarify that this only affects the message, so like - `<timezone_for_when_in_alert>` or something in that area.